magento/devdocs#: How to get RabbitMQ credentials/connection details (#5422)
magento/devdocs: guides/v2.2/cloud/project/project-conf-files_services-rabbit.md

1.  While the session is open, you can start a RabbitMQ client of your choice from your local workstation, configured to connect to the `localhost:<portnumber` using the username and password you found in the relationship variable. For this example, you would use `localhost:5672`.

> ### Get RabbitMQ credentials {#cloud-rabbitmq-credentials}

Reviewer: @atwixfirster   The information in this section is already provided in the [Service relationships topic](https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services.html#service-relationships).  That info could be improved in a separate PR.  For example, adding the additional command for listing the MAGENTO_CLOUD_RELATIONSHIPS variable information to the 'Verify relationships in a remote environment' and clarifying that  "Verifying relationships" is also the way to get the connection information. 

In this topic,  I suggest replacing Step 4, with the following and adding the steps for listing the connection information.

4.  Retrieve the RabbitMq connection details and login credentials from the [`$MAGENTO_CLOUD_RELATIONSHIPS`]({{ page.baseurl }}/cloud/env/environment-vars_cloud.html):

    [Insert your steps 1 and 2]

Shorten the response part:

In the response, find the RabbitMQ information, for example:

```json
{
   "rabbitmq" : [
      {
         "password" : "guest",
         "ip" : "246.0.129.2",
         "scheme" : "amqp",
         "port" : 5672,
         "host" : "rabbitmq.internal",
         "username" : "guest"
      }
   ]
}
```

Author: Thank you for your review, @meker12 

**implemented**

Could you please verify.

Thank you!

We use the {{site.data.var.ece}} environment variable [`$MAGENTO_CLOUD_RELATIONSHIPS`]({{ page.baseurl }}/cloud/env/environment-vars_cloud.html), a JSON object, to retrieve environment-related relationships.

To verify this information used for configurations and settings:

> 1. SSH into the Integration environment with RabbitMQ installed and configured.

Reviewer 2: Please, adjust this part to make the explanation more unified for all environments
```suggestion
1. SSH into your Magento Cloud environment where RabbitMQ is installed and configured.
```

Author: **implemented**

Thank you, @rogyar
